### ISSUE TITLE
There is no space after GitBranch

### ISSUE THREAD

**Issue by huyvohcmc** [glepnir/galaxyline.nvim#192]
I use the config in ![example/eviline.lua](https://github.com/glepnir/galaxyline.nvim/blob/main/example/eviline.lua) and noticing that there is no space after the GitBranch section:

<img width="972" alt="Screen Shot 2021-07-04 at 1 48 53 PM" src="https://user-images.githubusercontent.com/17645203/124375912-9f5d5600-dcce-11eb-9cf9-7009e2537a5d.png">

Also, there is an extra space before GitIcon and extra spaces between LineColumn, LinePercent as well.

**Comment by Tinku10** [comment]
@huyvohcmc  Yes, there are indeed some odd spacing in several places. More importantly spaces are not consistent. I also raised a similar issue https://github.com/glepnir/galaxyline.nvim/issues/199#issue-945202359. 

I was able to fix this issue locally though, you can have a look at my config attached with the issue.
